biolearn/data_library.py

```diff
@@ -6,6 +6,8 @@
 import gzip
 import shutil
 import os
+import matplotlib.pyplot as plt
```

Comment:
I don't think matplotlib is currently in the biolearn dependencies. We should add to pyproject.yaml